scripts/genModuleProvider.hs

```diff
@@ -38,26 +38,26 @@ import Language.Haskell.TH
 
 parse#{capModname} :: FromJSON a => FakerSettings -> Value -> Parser a
 parse#{capModname} settings (Object obj) = do
-  en <- obj .: (getLocale settings)
+  en <- obj .: (getLocaleKey settings)
```

Comment:
I actually did this PR by hand (lots of find and replace), so it might be good to rerun these scripts and make sure I didn't break anything. How do I rerun the scripts?

Reply:
I would worry too much about these scripts. And it's fine if they don't work, I can fix it in the next release. But If you are interested, I think I have some documentation on `DEVELOPMENT.md` on how I use them.